src/components/Tokens/state.ts

```diff
@@ -47,3 +47,9 @@ export function useSetSortCategory(category: Category) {
     }
   }, [category, sortCategory, setSortCategory, sortDirection, setDirectionCategory])
 }
+
+export function useIsFavorited(tokenAddress: string) {
+  const favoritedTokens = useAtomValue<string[]>(favoritesAtom)
+
+  return useMemo(() => favoritedTokens.includes(tokenAddress.toLocaleLowerCase()), [favoritedTokens, tokenAddress])
```

Comment:
one thing that will only impact us is whether someone has favorited before you forced lowercase, in which case `favoritedTokens` may have some non lowercased values. because we haven't launched i think this is okay, but in general you just want to be mindful of cached state

Reply:
ah good call out, didn't think of that.  If this pops up again, I'll keep this potential edge case in mind